src/cpp/src/whisper_pipeline.cpp

```diff
@@ -47,25 +48,21 @@ namespace genai {
 
 class WhisperPipeline::WhisperPipelineStatefulImpl : public WhisperPipeline::WhisperPipelineImplBase {
 public:
-    ov::genai::WhisperInitializedModels m_models;
+    ov::InferRequest m_encoder;
+    std::shared_ptr<ov::genai::WhisperDecoder> m_decoder;
```

Comment:
should these fields be moved to `private` section?

Reply:
Yes, sure, moved to private